.github/workflows/release.yml

```
@@ -26,8 +26,19 @@ jobs:
           cache: npm
           registry-url: https://registry.npmjs.org
 
+      - name: Get package version
+        run: |
+          VERSION=$(npm pkg get version | xargs)
+          echo "Package version is $VERSION"
+          echo "::set-output name=version::$VERSION"
```

```suggestion
          echo "name=$VERSION" >> $GITHUB_OUTPUT
```
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

---

Good catch.